modules/internal/ChapelAutoAggregation.chpl

```diff
@@ -87,9 +87,10 @@ module ChapelAutoAggregation {
     use CPtr;
     use AggregationPrimitives;
 
+    param defaultBuffSize = if CHPL_COMM == "ugni" then 4096 else 8096;
```

Comment:
This could potentially hurt performance in older infinibands, is that right? [.... but you can always fiddle with the environment variables to get better performance].

I am mostly curious, and fine with this part as-is

Reply:
Nah, if anything this is more impactful on older InfiniBand systems.

Reply:
Ah, right. I expect the older ones to be worse at handling many+smaller messages. I got confused, sorry.